core/renderers/zelos/constants.js

```diff
@@ -515,15 +516,77 @@ Blockly.zelos.ConstantProvider.prototype.makeRounded = function() {
   };
 };
 
+/**
+ * @return {!Object} An object containing sizing and path information about
+ *     a squared shape for connections.
+ * @package
+ */
+Blockly.zelos.ConstantProvider.prototype.makeSquared = function() {
+  var radius = this.CORNER_RADIUS;
+
+  // The 'up' and 'down' versions of the paths are the same, but the Y sign
+  // flips.  Forward and back are the signs to use to move the cursor in the
+  // direction that the path is being drawn.
```

Comment:
Could you rephrase this comment? I'm not entirely sure what this part means: Forward and back are the signs to use